dev/src/v1beta1/firestore_client.ts

```diff
@@ -52,6 +54,7 @@ const version = require('../../../package.json').version;
  *       committed. Any read with an equal or greater `read_time` is guaranteed
  *       to see the effects of the transaction.
  * @class
+ * @deprecated Use v1/firestore_client instead.
```

Comment:
Do we know specifically which tools/situations where this will be surfaced?  I think the TypeScript compiler, and eslint will do stuff.  Should we consider emitting runtime warnings for users not using these kinds of tools though?

Reply:
It shows up in the generated docs (barely legible) and as a warning in VSCode/IntelliJ/